controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -128,6 +128,9 @@
     public static final Property<String> PROPERTY_TLS_ENABLED_FOR_SEGMENT_STORE = Property.named(
             "segmentstore.connect.channel.tls", "", "auth.segmentStoreTlsEnabled");
 
+    public static final Property<Integer> PROPERTY_SEGMENT_STORE_REQUEST_TIMEOUT_MINUTES = Property.named(
+            "segmentstore.connect.channel.timeoutMinutes", 2, "");
```

Comment:
I would suggest you express this in seconds. We may want to add 30s timeouts. 2 minutes seems too long.